client/my-sites/checkout/composite-checkout/lib/filter-appropriate-payment-methods.ts

```diff
@@ -9,9 +9,24 @@ import type { PaymentMethod, LineItem } from '@automattic/composite-checkout';
  */
 import { CheckoutCartItem } from '../types/checkout-cart';
 import { CheckoutPaymentMethodSlug } from '../types/checkout-payment-method-slug';
+import config from 'calypso/config';
 
 const debug = debugFactory( 'calypso:composite-checkout:filter-appropriate-payment-methods' );
 
+const redirectPaymentMethods = [
```

Comment:
What do you think about defining a function called (for example) `isRedirectPaymentMethod( slug: CheckoutPaymentMethodSlug ) : boolean` in https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/checkout/composite-checkout/types/checkout-payment-method-slug.ts? That way any module that imports that type will also be able to detect redirect methods.

Reply:
Oh, I like this idea.

Reply:
In that case, this will be handled by the myriad changes proposed in https://github.com/Automattic/wp-calypso/pull/47215, but I need to split up that PR as it's doing too many things.